tck/src/main/java/org/eclipse/microprofile/config/tck/ConfigAccessorTest.java

```diff
@@ -196,6 +196,21 @@ public void testCacheFor() throws Exception {
         Assert.assertEquals(val.getValue(), "secondvalue");
     }
 
+    @Test
```

Comment:
This test seems weird, what should it actually do?

Reply:
the test checks that when the config source change the property value, the config is properly notified and the value from the ConfigAccessor is invalidated and fetch from the config source again.

I should have put a longer cache duration to make it clearer.